core/frontend/src/RealityDataSource.ts

```diff
@@ -5,15 +5,48 @@
 /** @packageDocumentation
  * @module Tiles
  */
-import { request, RequestOptions } from "./request/Request";
-import { AccessToken, BentleyStatus, GuidString, Logger } from "@itwin/core-bentley";
-import { IModelError, OrbitGtBlobProps, RealityData, RealityDataFormat, RealityDataProvider, RealityDataSourceKey, RealityDataSourceProps } from "@itwin/core-common";
+import { GuidString, Logger } from "@itwin/core-bentley";
+import { Cartographic, EcefLocationProps, OrbitGtBlobProps, RealityData, RealityDataFormat, RealityDataProvider, RealityDataSourceKey } from "@itwin/core-common";
 import { FrontendLoggerCategory } from "./FrontendLoggerCategory";
-import { IModelApp } from "./IModelApp";
-import { CesiumIonAssetProvider, ContextShareProvider, getCesiumAccessTokenAndEndpointUrl, getCesiumAssetUrl, getCesiumOSMBuildingsUrl } from "./tile/internal";
+import { CesiumIonAssetProvider, ContextShareProvider, getCesiumAssetUrl } from "./tile/internal";
+import { RealityDataSourceTilesetUrlImpl } from "./RealityDataSourceTilesetUrlImpl";
+import { RealityDataSourceContextShareImpl } from "./RealityDataSourceContextShareImpl";
+import { RealityDataSourceCesiumIonAssetImpl } from "./RealityDataSourceCesiumIonAssetImpl";
+import { Range3dProps } from "@itwin/core-geometry";
 
-/**
- * This interface provide methods used to access a reality data from a reality data provider
+const loggerCategory: string = FrontendLoggerCategory.RealityData;
+
+/** This interface provide spatial location and volume of interest, in meters, centered around `spatial location`
+ * @alpha
+ */
+export interface SpatialLocationAndExtents {
+  /** location of the point at the center of the reaity data */
+  location: Cartographic | EcefLocationProps;
```

Comment:
Is this interface a "props" (JSON) type or not? It mixes EcefLocationProps and Range3dProps, which are JSON types; with Cartographic, which is a class type.
It seems to me it should use EcefLocation and Range3d instead.

Reply:
SpatialLocationAndExtents is not a Props type. Change member to use class type instead of props.